docs/website/blog/2023-01-30-local-container-dev-on-podman.md

```diff
@@ -146,9 +146,8 @@ If you skipped Step 0 and selected "starter project", your output will be slight
 ### Step 2. Enabling the experimental mode
```

Comment:
Can we remove this section entirely now?

Reply:
I would prefer not to, by changing only the strict minimum the blog posts (here, we need to remove the links to the removed experimental-mode page, to make the website build)